src/components/ShowComments.jsx

```diff
@@ -0,0 +1,11 @@
+const ShowComments = ({ comment }) => {
+  return (
+    <div>
```

Comment:
Should this be a div, please be mindful of semantic html

Reply:
oh right, will change accordingly